src/server/kiwix-serve.cpp

```diff
@@ -715,13 +716,25 @@ static struct MHD_Response* handle_catalog(RequestContext* request)
       try {
         tags.push_back(request->get_argument("tag"));
       } catch (...) {}
+      try {
+        std::string excludeTags = request->get_argument("notag");
+        size_t start = 0;
+        for (size_t i = 0; i < excludeTags.size(); i++) {
+            if (excludeTags[i] == ';') {
+                noTags.push_back(excludeTags.substr(start, i - start));
+                start = i + 1;
+            }
+        }
```

Comment:
There is a function in kiwix-lib to split a string.
Something like this should works :
```
noTags = kiwix::split(request->get_argument("notag"), ";");
```

It could be nice to have the same thing for the accepted tags (in another commit)

Reply:
@mgautierfr Could you please "finish" this PR please as @jetownfeve21 won't be available in the next few weeks (at least)?